docs/reference/README.md

```diff
@@ -926,6 +926,24 @@ pronounce "tomato" as "<phoneme alphabet="ipa" ph="/təˈmɑːtoʊ/">tomato</pho
 Note that you must use SSML in the replacement text; the Litexa
 SSML shorthand statements (e.g "<!Bravo>") will not work.
 
+### localized pronunciations
+
+If you plan to publish your skill to multiple locales, you can define
+locale-specific pronunciations for your voiced responses. Simply add
+your locale-specific pronunciation definitions to the Litexa files that
+reside in said locale's Litexa project directory.
+
+::: tip Use a pronounce.litexa file in each locale's Litexa project directory
+If defining locale-specific pronunciations, we recommend creating a
+pronounce.litexa file in each of your locale's Litexa project directories.
+Put all of your locale-specific pronunciation definitions inside of it.
+:::
+
+::: warning The default locale's pronunciations do not carry over to other locales
+Because pronunciations are very locale-specific, the default locale's pronunciations
```

Comment:
would put `default` in backticks to make it more noticeable, and caveat this sentence with "Unlike the structure-based override features of localization, ..." since this is the comparison we're making.